x11-misc/sddm/sddm-0.19.0_p20201230.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+COMMIT=5825c89444626755ca7ce5eef454c4888131a12b
+inherit cmake linux-info pam systemd tmpfiles
+
+DESCRIPTION="Simple Desktop Display Manager"
+HOMEPAGE="https://github.com/sddm/sddm"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-2+ MIT CC-BY-3.0 CC-BY-SA-3.0 public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+IUSE="+elogind kernel-randomness openrc-init systemd test"
+
+REQUIRED_USE="^^ ( elogind systemd )"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	dev-python/docutils
+	dev-qt/linguist-tools:5
+	kde-frameworks/extra-cmake-modules:5
+	virtual/pkgconfig
+"
+COMMON_DEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	sys-libs/pam
+	x11-base/xorg-server
+	x11-libs/libxcb[xkb]
+	elogind? ( sys-auth/elogind )
+	systemd? ( sys-apps/systemd:= )
+	!systemd? ( sys-power/upower )
+"
+DEPEND="${COMMON_DEPEND}
+	test? ( dev-qt/qttest:5 )
+"
+RDEPEND="${COMMON_DEPEND}
+	acct-group/sddm
+	acct-user/sddm
+	!kernel-randomness? ( || (
+		sys-apps/haveged
+		sys-apps/rng-tools
+	) )
+"
+
+PATCHES=(
+	# Pending upstream
+	# fix for groups: https://github.com/sddm/sddm/issues/1159
+	"${FILESDIR}"/${PN}-0.19.0-revert-honor-PAM-supplemental-groups.patch
+	"${FILESDIR}"/${PN}-0.18.1-honor-PAM-supplemental-groups-v2.patch
+	# Downstream patches
+	"${FILESDIR}"/${PN}-0.18.1-respect-user-flags.patch
+	"${FILESDIR}"/${PN}-0.19.0-Xsession.patch # bug 611210
+	"${FILESDIR}"/${PN}-0.19.0-pam-examples.patch
+	"${FILESDIR}"/${PN}-0.19.0-no-pam_tally2.patch # PAM-1.4 woes, bug 728550
+)
+
+pkg_setup() {
+	local CONFIG_CHECK="~DRM"
+	use kernel-randomness && CONFIG_CHECK+=" ~RANDOM_TRUST_CPU"
+	use kernel_linux && linux-info_pkg_setup
+
+	if [[ -f "${EROOT}"/etc/sddm.conf ]]; then
+		cp -v "${EROOT}"/etc/sddm.conf "${T}"/sddm.conf.old || die
+	fi
+}
+
+src_prepare() {
+	touch "${S}"/01gentoo.conf || die
+
+cat <<-EOF >> "${S}"/01gentoo.conf
+[General]
+# Halt/Reboot command
+HaltCommand=$(usex elogind "/bin/loginctl" "/usr/bin/systemctl") poweroff
```

Comment:
`systemctl` might live in `/bin` depending on `USE=split-usr`.

Do these directives accept bare command names for a `PATH` lookup? If so, can you just do this instead?
```
HaltCommand=$(usex loginctl systemctl) poweroff
```